src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -1566,6 +1576,13 @@ static ucs_status_t uct_ib_mlx5_devx_xgvmi_umem_mr(uct_ib_mlx5_md_t *md,
     size_t length;
     void *mkc;
 
+
+    if (memh->dm != NULL) {
```

Comment:
Or `UCT_IB_MD_NO_DM`?

Reply:
but I have access to `memh` as opposed to `uct_ib_reg_mr`